src/main/java/uk/gov/hmcts/reform/cpo/controllers/CasePaymentOrdersController.java

```diff
@@ -65,14 +77,102 @@ public CasePaymentOrdersController(CasePaymentOrdersService casePaymentOrdersSer
         final List<String> listOfIds = ids.orElse(Collections.emptyList());
         final List<String> listOfCasesIds = casesId.orElse(Collections.emptyList());
         final CasePaymentOrderQueryFilter casePaymentOrderQueryFilter = CasePaymentOrderQueryFilter.builder()
-            .listOfIds(listOfIds)
-            .listOfCasesIds(listOfCasesIds)
-            .pageNumber(pageNumber.orElse(Integer.parseInt(applicationParams.getDefaultPageNumber())))
-            .pageSize(pageSize.orElse(Integer.parseInt(applicationParams.getDefaultPageSize())))
-            .build();
+                .listOfIds(listOfIds)
+                .listOfCasesIds(listOfCasesIds)
+                .pageNumber(pageNumber.orElse(Integer.parseInt(applicationParams.getDefaultPageNumber())))
+                .pageSize(pageSize.orElse(Integer.parseInt(applicationParams.getDefaultPageSize())))
+                .build();
         return casePaymentOrdersService.getCasePaymentOrders(casePaymentOrderQueryFilter);
     }
 
+    @DeleteMapping(path = CASE_PAYMENT_ORDERS_PATH, params = IDS)
+    @ResponseStatus(HttpStatus.OK)
+    @ApiOperation(value = "Delete case payment orders by id", notes = "Delete case payment orders by id")
+    @ApiResponses({
+        @ApiResponse(
+            code = 200,
+            message = ""
+        ),
+        @ApiResponse(
+            code = 400,
+            message = ValidationError.IDS_EMPTY,
+            examples = @Example(value = {
+                @ExampleProperty(
+                    value = "{\n"
+                        + "   \"status\": \"BAD_REQUEST\",\n"
+                        + "   \"errors\": [\n"
+                        + "   \"message\": \"" + ValidationError.IDS_EMPTY + "\",\n"
+                        + "   ]\n"
+                        + "}",
+                    mediaType = APPLICATION_JSON_VALUE
+                )
+            })
+        ),
+        @ApiResponse(
+            code = 401,
+            message = AuthError.AUTHENTICATION_TOKEN_INVALID
+        ),
+        @ApiResponse(
+            code = 403,
+            message = AuthError.UNAUTHORISED_S2S_SERVICE
+        )
+    })
+    public void deleteCasePaymentOrdersById(@RequestParam(IDS) @Valid
+                                                  @NotEmpty(message = ValidationError.IDS_EMPTY)
+                                                          List<UUID> ids)
```

Comment:
Would you get more validation control (ie.. better messages) by using a list of strings and `@ValidCpoId`?

Reply:
Now refactored to mirror `GET` endpoint functionality